pages/conduct.tsx

```diff
@@ -0,0 +1,119 @@
+import Head from 'next/head';
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer';
+import { Box, Flex, Heading, Image, Text } from '@chakra-ui/react';
+
+import Layout from '../components/Layout';
+import { GenericContentfulPageInterface } from '../types';
+
+export default function ConductPage({
+  fields,
+  children,
+}: GenericContentfulPageInterface) {
+  const {
+    title,
+    description,
+    subtitle1,
+    contentBlock1,
+    subtitle2,
+    contentBlock2,
+    images,
+    ogDescript,
+  } = fields;
+
+  const genericImages = images?.map((image) => image.fields);
+
+  return (
+    <Layout>
+      <Head>
+        <meta property="og:title" content={title} key="ogtitle" />
+        {ogDescript && (
+          <meta property="og:description" content={ogDescript} key="ogdesc" />
+        )}
+        <title>Code of Conduct</title>
+      </Head>
+
+      <Flex direction="column" align="center" bg="blackAlpha.200" py="2rem">
+        <Heading as="h1" size="lg" textAlign="center" my={2}>
+          Code of Conduct
+        </Heading>
+        <Text fontSize="lg" my="1rem" w={[350, 550, 700, 800]} px="1rem">
```

Comment:
I've added you to the contentful org if you want to run these through that - otherwise as mentioned I'll migrate them over when I get time